docs/get-started/whats-new.md

```diff
@@ -0,0 +1,17 @@
+# ReactMapGL v3.0
+
+Release date: TBD
+
+## Highlights
```

Comment:
These are the highlights, and things that maybe the user would be interested in seeing first. I'm planning on adding more details to the doc with all/most of the smaller changes. But feel free to add anything you think is missing!